src/components/article/Image.js

```diff
@@ -1,22 +1,26 @@
 'use strict'
 import _ from 'lodash'
 import classNames from 'classnames'
-import FitwidthMixin from '../../lib/FitwidthMixin'
+import FitwidthMixin from '../mixins/FitwidthMixin'
 import { getScreenType } from '../../lib/screen-type'
 import BlockAlignmentWrapper from './BlockAlignmentWrapper'
 import React, { Component } from 'react'
 import LazyLoad from 'react-lazy-load'
 import styles from './Image.scss'
 import UI_SETTING from '../../constants/ui-settings'
 
+const DEFAULT_WIDTH = 200
+const DEFAULT_HEIGHT = 200
+const DEFAULT_PLACEHOLDER = 'https://cdn-images-2.medium.com/freeze/max/30/1*HKrv5OV9P63vz5sa8-Cceg.png?q=20'
```

Comment:
I didn't change the default image placeholder.
May need @garfieldduck  to explain this?